src/gg.regression.unity.bots/Runtime/Scripts/RGHeadlessSequenceRunner.cs

```diff
@@ -30,9 +30,12 @@ public static class RGHeadlessSequenceRunner
         [RuntimeInitializeOnLoadMethod(RuntimeInitializeLoadType.BeforeSplashScreen)]
         public static void FirstLoadChecks()
         {
-            // do this here to fail fast on bad args
-            ParsePathArgument();
-            ParseTimeoutArgument();
+            // If we are not in headless mode, we don't need these checks.
+            if (Application.isBatchMode){
```

Comment:
I can also use our IsBatchMode() here, or convert our IsBatchMode() usage to this. 

Let me know which you prefer.

Reply:
This isn't the right fix; updated fix pushed

- These options are supported whether the runtime is in batch mode or not
- The correct fix is in `ParseTimeoutArgument` where we should only validate the path IF timeout is supplied